CameraApp0/PreSetup

```diff
@@ -1,5 +1,5 @@
 #!/bin/bash
 
 pip install notebook
```

Comment:
The version of `notebook` is not fixed. This has a potential runtime error in the long run.
Freezing versions would resolve this kind of errors.
For example, you can add `requirements.txt` and run `pip install -r requirements.txt` in PreSetup script.

Reply:
Thanks! Update as you recommended.